app/views/editor/posts/index.html.haml

```diff
@@ -21,6 +21,7 @@
         %td= post.author&.name
         %td= image_tag post.thumbnail_url, height: 80 if post.thumbnail.present?
         %td= link_to 'Show', editor_post_url(post)
+        %td= link_to 'Preview', preview_editor_post_url(post)
```

Comment:
An editor can't be back from preview page to editor page. So it is kindness that `target: '_blank' is specified.